pkg/populator/oidc.go

```diff
@@ -1,62 +1,103 @@
 package populator
 
 import (
-	"os"
+	"fmt"
 	"sync/atomic"
 
+	"github.com/nerdalize/nerd/nerd/conf"
+
+	v1payload "github.com/nerdalize/nerd/nerd/client/auth/v1/payload"
 	"github.com/pkg/errors"
 	"k8s.io/client-go/tools/clientcmd/api"
+	// this blank import is necessary to load the oidc plugin for client-go
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
```

Comment:
This plugin is important, otherwise there is an error `No Auth Provider found for name "oidc"`
https://github.com/kubernetes/client-go/issues/345